commands/scanpullrequest.go

```diff
@@ -56,7 +61,15 @@ func scanPullRequest(params *utils.FrogbotParams, client vcsclient.VcsClient) er
 	// Comment frogbot message on the PR
```

Comment:
1. In the description of this fucntion, we should change - 
Scan a pull request by as follows: --> Scan a pull request as follows: 

2. In addition, the description ignores the scenario where Frogbot is configured to show all vulnerabilities. 

3. Also, I suggest improving the "Include all vulnerabilities scan is on" log message in this function and changing it to - "Frogbot is configured to show all vulnerabilities"